### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by ryanseys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +189,7 @@ Saves the provided stats.
 - **GitStats** The `GitStats` object.
 
 ### `iterateDays(data, callback)`
-Iterate the days, calling the callback function for each day.
+Iterate through the days, calling the callback function on each day.
```
Comment (IonicaBizau) [inline README.md]: Please update here the JSDoc comment too: https://github.com/IonicaBizau/git-stats/blob/59884d52c52f084a6bf52fd362ee19f64bdfeaef/lib/index.js#L133

**Comment by IonicaBizau** [comment]
Hey, you are @ryanseys from @Google! Glad to see you in this place and I am honored to merge this PR into master!

I remember we had some conversations in the google-api-nodejs-client repository. :smile:

**Comment by ryanseys** [comment]
Yes, I came across this repo and it's totally awesome! I figured I'd help out in the usual way I first do with most projects, by contributing to the README. I love how you've created many different modules for different uses and how they all work together to make this module more useful :)

Are your README docs generated from your JSDocs? Or do I need to update both?

**Comment by IonicaBizau** [comment]
Awww, thanks! :blush: 

Since you already updated the README file, just update the JSDoc comment and that's fine.

File: README.md (hunk 1 above)
Comment (ryanseys) [inline README.md]: Fixed.

**Comment by IonicaBizau** [comment]
Merged. BTW, this is the first PR I accept from a Google employee! :sunglasses:

**Comment by ryanseys** [comment]
:+1: Too bad it wasn't a more technical PR. I love good documentation though. :smile:

**Comment by IonicaBizau** [comment]
I also like good docs. And I'm not an English speaker, that's why I love these pull requests. :smile: 

I use [`blah`](https://github.com/IonicaBizau/node-blah) for generating docs.

**Comment by ryanseys** [comment]
You fooled me, they just looked like typos to me. :) `blah` looks neat too.

**Pull request merged**